command/archive_tarball_create.go

```diff
@@ -85,14 +91,23 @@ func (c *tarballCreate) Execute(ctx context.Context,
 		}()
 		var err error
 		filesArchived, err = c.makeArchive(ctx, logger.Execution())
-		logger.Execution().Debugln("Finished making archive")
-
 		errChan <- errors.WithStack(err)
 	}()
 
 	select {
 	case err := <-errChan:
 		if err != nil {
+			// this is only our first attempt, so we should retry if we've hit a relevant error
+			if c.Attempt == 0 {
```

Comment:
Should this be a bool? It should only ever be 0 or 1, which suggests false or true.

Reply:
I thought about that, but figured it could leave the implementation open for us to specify a max retries that's greater than 1. Maybe I'll just reword to use maxRetries but set that to 1.